packages/cozy-pouch-link/src/CozyPouchLink.js

```diff
@@ -122,7 +123,9 @@ class PouchLink extends CozyLink {
 
     /** @private */
     this.startReplicationDebounced = debounce(
-      this._startReplication,
+      options => {
+        return this._startReplication(options)
+      },
```

Comment:
This change seems to be equivalent to previous code, is it necessary?

Reply:
Not really, I changed it for testing, and I kind of prefer explicit code. I reverted it